src/frontends/onnx/frontend/src/op/transpose.cpp

```diff
@@ -4,11 +4,10 @@
 
 #include "op/transpose.hpp"
 
-#include <memory>
-#include <vector>
-
 #include "ov_models/ov_builders/reshape.hpp"
 
+using namespace ov::op;
```

Comment:
With this `using namespace`, we can reduce redundant `ov::op::util::transpose(data)` to `util::transpose` in line 21, for example